src/Compatibility/ApiDiff/Microsoft.DotNet.ApiDiff/DiffConfiguration.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.DotNet.ApiDiff;
+
+/// <summary>
+/// Defines a variety of configuration options for API diff.
+/// </summary>
+public class DiffConfiguration
```

Comment:
Looks a good example of a POCO. Should this be a record to be more lightweight?

Reply:
Done.